README.md

```diff
@@ -108,6 +107,11 @@ The configuration of benchmarks allows you to select the frameworks to run, sele
 |**[GradientBoostingClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingClassifier.html)**|gbt|:x:|:x:|:x:|:white_check_mark:|
 |**[GradientBoostingRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.GradientBoostingRegressor.html)**|gbt|:x:|:x:|:x:|:white_check_mark:|
 
+## Intel(R) Extension for Scikit-learn* support
+[Intel(R) Extension for Scikit-learn](https://github.com/intel/scikit-learn-intelex) providing drop-in patching speeds up scikit-learn on CPU and GPU. 
+
+Scikit-learn benchmarks on GPU with Intel(R) Extension for Scikit-learn support algorithms: **DBSCAN**, **KMeans**, **LinearRegression**, **LogisticRegression**. Example config with GPU support [here](https://github.com/IntelPython/scikit-learn_bench/blob/master/configs/skl_xpu_config.json).
```

Comment:
The following benchmarks support GPU computations with help of Intel(R) Extension for Scikit-learn*:
- dbscan
- kmeans ...

Reply:
Example config with GPU support [here]: The configuration file that contains all of these benchmarks can be found [here].
The link for "here" is broken.
Please add an example of command how to run this config on GPU here